xblock/test/django/test_request.py

```diff
@@ -80,12 +80,12 @@ def test_headers(self):
     def test_content_type(self):
         # Default charset
         self.assertEquals(
-            self._as_django(content_type='application/json')['Content-Type'],
+            self._as_django(content_type='application/json; charset=UTF-8')['Content-Type'],
```

Comment:
`_as_django` takes the charset and content type separately so that the caller doesn't have to worry about the correct format of the header. Can we leave the signature as it was?

Also, as I mentioned here: https://github.com/edx/XBlock/pull/337/files, I think that this test should just validate that specifying `content_type` gives you the content type you asked for (without asserting anything about the charset), and that setting both gives you a header that has both in it.